rpc/send.go

```diff
@@ -125,28 +124,18 @@ func Send(opts Options, method string, addrs []net.Addr, getArgs func(addr net.A
 		}
 	}
 
-	var clients []*Client
+	if opts.Timeout == 0 {
+		// Wait forever.
+		opts.Timeout = math.MaxInt64
+	}
+
+	var orderedAddrs []net.Addr
 	switch opts.Ordering {
 	case OrderStable:
-		for _, addr := range addrs {
-			clients = append(clients, NewClient(addr, nil, context))
-		}
+		orderedAddrs = addrs
 	case OrderRandom:
-		// Randomly permute order, but keep known-unhealthy clients last.
```

Comment:
`IsHealthy` went away. Is the following code cheap? If so, might be a good way to re-implement it. I'm also wondering whether we really need `c.isHealthy` (a bool) since we only sequentially touch the channel from a single function which can easily remember whether the channel was closed or not.
```
select {
  <-c.Healthy():
     return true
}
return false
```